aod-discord-bot.js

```diff
@@ -319,6 +322,20 @@ function messageReply(message, msg, edit) {
 }
 global.messageReply = messageReply;
 
+function sendGlobalNotification(interaction, msg) {
```

Comment:
Might seem a bit pointless, but the convention in the bot code is to use message. When an interaction is processed, a boolean isInteraction is added to let the function hand any differences.

Reply:
Resolved in d507beb